### ISSUE TITLE
Add support for categories

### ISSUE THREAD

**Issue by ymartin59** [SynoCommunity/spkrepo#37]
Package Center now displays Synology packages organized as "Categories".
It may be useful to publish SynoCommunity the same way.

**Comment by mreid-tt** [comment]
Looking into this a bit with the Synology packages, the category tag seems to be associated with the JSON result from the query on the Package Center server and not associated with the package itself (i.e. doesn't seem to be in the package INFO file).

As such, if this is a feature we still want then we would need to setup the various categories on the server database and at the package level in the admin interface, allow for the tagging of one or more category identifiers. With this present the JSON result to the client could include this data.

Is this something we should still explore? Have we seen examples of other Third-Party Package providers which show categories? I'm wondering if the Package Center will present the categories in the same way that the Synology packages are presented.

**Comment by hgy59** [comment]
@mreid-tt if we investigate into categories, we should first validate whether categories work for thirdparty packages in the DSM Package Center.

**Comment by mreid-tt** [comment]
> @mreid-tt if we investigate into categories, we should first validate whether categories work for thirdparty packages in the DSM Package Center.

I completely agree. No point in extending our database if DSM does not display categories for community packages.
